app/api/attendees.py

```diff
@@ -118,12 +118,15 @@ def before_update_object(self, obj, data, kwargs):
             raise ForbiddenException({'source': 'User'}, 'You are not authorized to access this.')
 
         if 'is_checked_in' in data:
-            if data['is_checked_in'] and 'checkin_times' not in data:
-                raise UnprocessableEntity({'pointer': '/data/attributes/checkin_times'},
-                                          "Check in time missing while trying to check in attendee")
-
-            if obj.checkin_times and data['checkin_times'] not in obj.checkin_times.split(","):
-                data['checkin_times'] = '{},{}'.format(obj.checkin_times, data['checkin_times'])
+            if data['is_checked_in']:
```

Comment:
If in `data` `checkin_times` doesn't present they it will definitely throw KeyError. Try to handle this situation.

Reply:
[This line](https://github.com/fossasia/open-event-orga-server/pull/4489/files#diff-5e5ae0e99c073cd15ba3cd3e8bbf7c72R122) checks if `checkin_times` is present and handles the error

Reply:
This can easily go in a single single, like 
```
if 'is_checked_in' in data and data['is_checked_in']:
```